deployment/data-streams/changeset/state.go

```diff
@@ -80,8 +80,12 @@ func LoadChainState(logger logger.Logger, chain deployment.Chain, addresses map[
 	cc.Verifiers = make(map[common.Address]*verifier.Verifier)
 	cc.VerifierProxys = make(map[common.Address]*verifier_proxy_v0_5_0.VerifierProxy)
 
-	for address, tvStr := range addresses {
-		switch tvStr.String() {
+	for address, tv := range addresses {
+		if belongsToMCMS(address, mcmsWithTimelock) {
```

Comment:
I think we maybe still want to do this - we can call read only methods that don't require ownership

Reply:
Actually the MCMS contracts are loaded [here](https://github.com/smartcontractkit/chainlink/blob/feat/mcms-ds-cs-tests/deployment/data-streams/changeset/state.go#L69)
Other teams seems to just ignore [these](https://github.com/smartcontractkit/chainlink/blob/feat/mcms-ds-cs-tests/deployment/keystone/changeset/state.go#L306), but I believe using this function is better.

Reply:
I think I interpreted incorrectly